apps/web/src/pages/Auth/SignInPage/SignInPage.tsx

```diff
@@ -49,141 +57,554 @@ const SignInPage = () => {
+              })}
+            </p>
+            <p data-h2-margin="base(x.5, 0, 0, 0)">
+              {intl.formatMessage(
+                {
+                  defaultMessage:
+                    "<strong>Don't have a GCKey account?</strong> <a>Sign up for one</a>.",
+                  id: "QsWizb",
+                  description:
+                    "Instruction on what to do if user does not have a GCKey",
+                },
+                {
+                  a: (chunks: React.ReactNode) =>
+                    buildExternalLink(loginPath, chunks),
+                },
+              )}
             </p>
-          </div>
+            <p data-h2-margin="base(x1, 0, 0, 0)">
+              {intl.formatMessage({
+                defaultMessage:
+                  "<strong>Heads up!</strong> Before we send you off to set up your GCKey credentials, we ask that you take another look at the program's eligibility criteria to ensure that you meet the requirements.",
+                id: "ow71P8",
+                description:
+                  "A request for the user to review the IAP eligibility criteria",
+              })}
+            </p>
+            <p data-h2-margin="base(x2, 0, 0, 0)">
+              {intl.formatMessage(
+                {
+                  defaultMessage:
+                    "If you have questions concerning this step, or if you are unsure about how to proceed, please feel free to reach out to our support team at <a>edsc.pda-iap.esdc@hrsdc-rhdcc.gc.ca</a>.",
+                  id: "ltzA7w",
+                  description:
+                    "How to get help from the support team - IAP variant",
+                },
+                {
+                  a: (chunks: React.ReactNode) =>
+                    buildExternalLink(
+                      "mailto:edsc.pda-iap.esdc@hrsdc-rhdcc.gc.ca",
+                      chunks,
+                    ),
+                },
+              )}
+            </p>
+          </>
+        )}
+
+        <hr data-h2-margin="base(x2, 0)" />
+        <div
+          data-h2-display="base(flex)"
+          data-h2-flex-direction="base(column) l-tablet(row)"
+          data-h2-align-items="base(center)"
+          data-h2-justify-content="base(space-between)"
+          data-h2-gap="base(x1)"
+        >
+          <p>
+            <Link href={paths.register()} mode="inline">
+              {intl.formatMessage({
+                defaultMessage: "Sign up instead",
+                id: "aqkSW2",
+                description: "Link text to register instead of signing in",
+              })}
+            </Link>
+          </p>
+          <p>
+            <Link href={loginPath} mode="solid" color="primary" external>
+              {intl.formatMessage({
+                defaultMessage: "Continue to GCKey and sign in",
+                id: "4sLCWZ",
+                description: "GCKey sign in link text on the sign in page",
+              })}
+            </Link>
+          </p>
```

Comment:
Needs some padding between the two buttons on mobile view
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/22059495/31c94790-1db4-4790-9a9d-5b260d9b7d6f)

`data-h2-margin-bottom="base(x1)"`

Reply:
I think that was already done by Peter in https://github.com/GCTC-NTGC/gc-digital-talent/pull/7835/commits/8fabb396dca6bcbf0dc5e32332e2e73c3034efda
Some tweaks from the sister page PR were rolled into this one